modules/ui/fields/input.js

```diff
@@ -350,7 +350,7 @@ export function uiFieldText(field, context) {
         if (input.empty() || !Object.keys(_phoneFormats).length) return;
 
         var extent = combinedEntityExtent();
-        var countryCode = extent && countryCoder.iso1A2Code(extent.center());
+        var countryCode = extent && countryCoder.iso1A2Code(extent.center(), { level: 'territory' });
         var format = countryCode && _phoneFormats[countryCode.toLowerCase()];
```

Comment:
to fall back for cases where there is a _territory_, but the phone format is only defined for the _country_ level, consider something like this:

```suggestion
        let region = extent && countryCoder.feature(extent.center(), { level: 'territory' });
        let format = region && _phoneFormats[region.properties.iso1A2.toLowerCase()];
        if (region.properties.level === 'territory' && !format) {
            // format not specified for territory -> try again at country level
            region = extent && countryCoder.feature(extent.center(), { level: 'country' });
            format = region && _phoneFormats[region.properties.iso1A2.toLowerCase()];
        }
```

Reply:
Yeah something like that, I just haven't started yet.

Reply:
@tyrasd I can't seem to get to you, but as per https://github.com/openstreetmap/iD/pull/10904#issuecomment-2812559990 , this PR is simple, uncontroversial, and ready for review.